BasicSample/Assets/ARMesh/Scripts/MeshFollowUser.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.XR.ARFoundation;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    /// <summary>
+    /// Updates the mesh bounding volume every frame to follow the user's head.
+    /// </summary>
+    public class MeshFollowUser : MonoBehaviour
```

Comment:
nit:  consider to rename.  Considering the typical usage is to put this script side by side with the "ARMeshManager", How about calling this "ARMeshUpdateFollowsUser"?

Reply:
Updated!